cmd/crates/soroban-rpc/README.md

```diff
@@ -0,0 +1,3 @@
+# soroban-rpc
+
+Tools and utilities for soroban rpc.
```

Comment:
Would be ideal to have a more comprehensive README but this is fine to get the crate created/published